lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -435,32 +449,33 @@ def column_definitions(table_name)
             /interval/,
             /numeric/
           ]
+
           re = Regexp.union(target_types)
```

Comment:
is `target_types` and this `regexp` unused now?

Reply:
Yes it's in the condition on https://github.com/cockroachdb/activerecord-cockroachdb-adapter/pull/235/files#diff-aa6cbad03effcf32e5ea1d8867d8727eb6a605525828574bd3b2ef8c249f8f54R456

Reply:
missed that, thanks!